src/transformers/models/bert/modeling_bert.py

```diff
@@ -1184,7 +1183,6 @@ def set_output_embeddings(self, new_embeddings):
 
     @add_start_docstrings_to_model_forward(BERT_INPUTS_DOCSTRING.format("batch_size, sequence_length"))
     @add_code_sample_docstrings(
-        processor_class=_TOKENIZER_FOR_DOC,
         checkpoint=_CHECKPOINT_FOR_DOC,
```

Comment:
not sure why this is not added say as the last argument and same question below

Reply:
Because there is no `processor_class` in the code sample used by this model anymore. It's just in the base model, and I have just left it there in case it's used by other modalities.